rules/aip0121/resource_must_support_get.go

```diff
@@ -33,16 +33,16 @@ var resourceMustSupportGet = &lint.ServiceRule{
 		// Iterate all RPCs and try to find resources. Mark the
 		// resources which have a Get method, and which ones do not.
 		for _, m := range s.GetMethods() {
-			if utils.IsGetMethod(m) {
+			if utils.IsGetMethod(m) && utils.IsResource(utils.GetResponseType(m)) {
```

Comment:
This presumably won't don't oddities like streaming requests or responses, but it's tighter than it was.

Reply:
That's a good point, it won't. I might come back for that though, same goes for the Must have List rule as well.